requirements.txt

```diff
@@ -31,6 +31,7 @@ pywinrm
 PyYAML
 requests
 riggerlib
+Runner
```

Comment:
There are several libraries that do this, why make a new one and then not follow pep8?